demos/common/hfsm/az_iot_hfsm_sync_adapter.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file az_iot_hfsm_sync_adapter.h
+ * @brief Adapter for HFSM syncrhonous operation.
+ * 
+ * @details This adapter provides a way to convert from syncrhonous IoT Provisioning and Hub
```

Comment:
convert from syncrhonous -> convert from synchronous

Reply:
Not sure why but my spell checker isn't picking that up...